contracts/MetadataViews.cdc

```diff
@@ -20,6 +20,11 @@ pub contract MetadataViews {
     pub resource interface Resolver {
         pub fun getViews(): [Type]
         pub fun resolveView(_ view: Type): AnyStruct?
+
+        /// Borrow the subcollection if this resolver has any sub nfts in it
+        pub fun borrowSubCollection() : &{ResolverCollection}? {
+            return nil
+        }
```

Comment:
What if there are multiple collections?

Reply:
Good question, should this method have a classifier? if it does then we need to have another method to actually get what subCollections you have.

Reply:
yeah, I don't know if we can standardize how to retrieve a specific collection. There could be different collection types stored, different collections of the same type differentiated by some random piece of metadata, or any other method.

It makes me think we should just have this return everything. Maybe we just return an array of collection references? or a dictionary that maps `{AnyStruct: &{ResolverCollection}?}` so the implementations can decide how they want to key them. Doesn't seem great either. :(